manifests/operator.yaml

```diff
@@ -220,10 +257,11 @@ spec:
       serviceAccountName: cockroach-operator-sa
       containers:
         - name: cockroach-operator
-          image: cockroachdb/cockroach-operator:v1.0.3
+          image: us.gcr.io/chris-love-operator-playground/cockroachdb-operator:v1.8.30-alpha.1
           imagePullPolicy: IfNotPresent
           # uncomment this to enable ParitionedUpdates
-          # args: ["-feature-gates","PartitionedUpdate=true,UseDecommission=true"]
+          # args: ["-feature-gates","PartitionedUpdate=true,UseDecommission=true,Upgrade=false, CrdbVersionValidator=false"]
+          args: ["-feature-gates", "CrdbVersionValidator=true"]
```

Comment:
@keith-mcclellan we probably need to just enable `CrdbVersionValidator` and `ParitionedUpdate` by default

Reply:
yes I agree this should be the default